Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -892,12 +892,8 @@ extension BrowserViewController {
           }
         }
         popup.addButton(title: Strings.openExternalAppURLAllow, type: .primary) { [weak tab] () -> PopupViewDismissType in
-          if UIApplication.shared.canOpenURL(url) {
-            UIApplication.shared.open(url, options: [:]) { didOpen in
-              openedURLCompletionHandler(!didOpen)
-            }
-          } else {
-            openedURLCompletionHandler(true)
+          UIApplication.shared.open(url, options: [:]) { didOpen in
```

Comment:
@soner-yuksel this should have probably been `open(url, options: [.universalLinksOnly: true])`?

Reply:
This prompt is for external URLs with non-https schemes is it not? Universal links wouldn't be associated with those anyways

Reply:
Yeah they are non-https schemes, next time we make a chanhe here. (I am sure it will happen:)) I will add this with it.